CHANGELOG.rst

```diff
@@ -2,6 +2,137 @@
 Release Notes
 =============
 
+.. _Release Notes_7.53.0:
+
+7.53.0 / 6.53.0
+======
+
+.. _Release Notes_7.53.0_Prelude:
+
+Prelude
+-------
+
+Release on: 2024-04-30
+
+- Please refer to the `7.53.0 tag on integrations-core <https://github.com/DataDog/integrations-core/blob/master/AGENT_CHANGELOG.md#datadog-agent-version-7530>`_ for the list of changes on the Core Checks
+
+
+.. _Release Notes_7.53.0_New Features:
+
+New Features
+------------
+
+- Support database-monitoring autodiscovery for Aurora cluster instances. Adds a new configuration listener to poll
+  for a specific set of Aurora cluster IDs and then create a new database-monitoring supported check
+  configuration for each endpoint. This allows for monitoring of endpoints that scale dynamically.
+
+- Add new core check orchestrator_ecs to collect running ECS tasks
+
+- APM stats now include an is_trace_root field to indicate if the stats are from the root span of a trace.
+
+- The cluster-agent now collects network policies from the cluster.
+
+- Enable 'host_benchmarks' by default when running the security-agent compliance module.
+
+- OTLP ingest now has a feature flag to identify top-level spans by span kind. This new logic can be enabled by adding `enable_otlp_compute_top_level_by_span_kind` in DD_APM_FEATURES.
+  - With this new logic, root spans and spans with a server or consumer `span.kind` will be marked as top-level. Additionally, spans with a client or producer `span.kind` will have stats computed.
+  - Enabling this feature flag may increase the number of spans that generate trace metrics, and may change which spans appear as top-level in Datadog.
+
+- Experimental: The process-agent checks (process, container, and process-discovery) can be run from the Core Agent in
+  Linux. This feature can be toggled on by setting the `process_config.run_in_core_agent.enabled` flag to `true` in
+  the `datadog.yaml` file. This feature is disabled by default.
+
+
+.. _Release Notes_7.53.0_Enhancement Notes:
+
+Enhancement Notes
+-----------------
+
+- Add the container image and container lifecycle checks to the output of the Agent status command.
+
+- Add `kubelet_core_check_enabled` flag to Agent config to control  
+  whether the kubelet core check should be loaded.
+
+- Added LastSuccessfulTime to cronjob status payload.
+
+- Add a retry mechanism to Software Bill of Materials (SBOM) collection for container images.
+  This will help to avoid intermittent failures during the collection process.
+
+- Add startup timestamp to the Agent metadata payload.
+
+- Agents are now built with Go ``1.21.8``.
```

Comment:
nit: remove this entry (semantically, it should be replaced by the next entry about Go 1.21.9)